app/src/main/res/layout/income_activity.xml

```diff
@@ -4,4 +4,4 @@
     android:id="@+id/container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".income" />
+    tools:context=".Income" />
```

Comment:
Change for income -> Income for proper class naming convention of UpperCamelCase